client/src/views/CodeView.vue

```diff
@@ -83,15 +84,28 @@ export default {
       language: "sql",
       editor: null,
       schema: "",
+      planId: null,
     };
   },
+  created: function () {
+    this.planId = this.$route.params.id;
+    if (this.planId) {
+      this.loadPlan(this.planId);
```

Comment:
Do we need to await or return loadPlan here?  Do we care if the view continues loading while the plan is being fetched?

Reply:
I am currently keeping the plan loading in the mounted stage, where it should run after the components are loaded.